framework/src/headers/ip/v6/mod.rs

```diff
@@ -382,4 +397,22 @@ mod tests {
         assert_eq!(ip.hop_limit(), 2);
         assert_eq!("2001:db8::1 > 2001:db8::2 version: 6 traffic_class: 17 flow_label: 15000 len: 1000 next_header: Udp hop_limit: 2", ip.to_string())
     }
+
+    proptest! {
+
+        //proptest generates the following format 128 bit ipv6 address # 1:2:3:4:5:6:7:8
+        #[test]
+        fn test_og_sender_not_og_prop_test(ref s in "([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{4}"){
```

Comment:
since this is not a og-specific side of the repo, probably worth renaming stuff on these.